client/app/pages/queries/schema-browser.html

```diff
@@ -1,13 +1,21 @@
 <div class="schema-container">
-  <div class="p-t-5 p-b-5">
```

Comment:
I'm curious about this change (`p-t-5 p-b-5` -> `schema-control`). It has obvious benefits, but it has the downside of making the CSS code larger. Any thoughts on this? When would you use a generic class vs. specific one? 

(this is just for discussion, no need to change anything)